sys/include/sched_round_robin.h

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (C) 2020 TUBA Freiberg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    sche_round_robin Round Robin Scheduler
+ * @ingroup     sys
+ * @brief       This module module provides round robin scheduling within
+ *              for all runnable threads within each not masked priority.
+ *              Priority 0 is masked by default.
+ *              There is no fainess involved in this scheuler. It schedules,
```

Comment:
```suggestion
 *              There is no fairness involved in this scheduler. It schedules,
```

Reply:
@kfessel Many editors can do spell checking. E.g. [the best editor](https://neovim.io/) can certainly do this ;-) Most are even smart enough to not spell check source code, but only comments and strings.

Reply:
i just enabled the spell check by default option